src/main/resources/static/api.yml

```diff
@@ -1717,6 +1717,35 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /applications/{applicationId}/withdraw:
+    put:
```

Comment:
I think this should be a `POST` to `/applications/{applicationId}/withdrawl` because we're creating a withdrawl here